src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const objValueOfDate = {};
+  const arrDateData = date.split(fromFormat[3]);
+  const newArrDate = [];
```

Comment:
The variable newArrDate is not descriptive. Consider renaming it to something more meaningful, like 'formattedDateParts'.